eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,18 +1,20 @@
 parameters:
   Artifacts: []
+  RunUnitTests: true
   ServiceDirectory: not-specified
 
 stages:
-- stage: Build
-  jobs:
-  - template: ../jobs/archetype-sdk-client.yml
-    parameters:
-      ServiceDirectory: ${{parameters.ServiceDirectory}}
+  - stage: Build
+    jobs:
+      - template: ../jobs/archetype-sdk-client.yml
+        parameters:
+          ServiceDirectory: ${{parameters.ServiceDirectory}}
 
-# The Prerelease and Release stages are conditioned on whether we are building a pull request and the branch.
-- ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
-  - template: pipelines/stages/archetype-js-release.yml@azure-sdk-build-tools
-    parameters:
-      DependsOn: Build
-      Artifacts: ${{parameters.Artifacts}}
-      ArtifactName: packages
+  # The Prerelease and Release stages are conditioned on whether we are building a pull request and the branch.
+  - ? ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}
```

Comment:
Why did you change this? Is there some interesting difference that we should pay attention to?

Reply:
I have a feeling this happened because of some styling/ formatting changes that get applied to when we save a file. I am seeing similar changes in my PR when I save my code in VS Code.

Reply:
Yep .. that is what it is. I guess if we are going to use these pretification tools we should be comfortable with the results :)

Reply:
Is that even valid syntax? I don't think I've seen that before or in the docs.